posts/2023-08-30_pharmaverse_story/pharmaverse_story.qmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "The pharmaverse (hi)story"
+author:
+  - name: Nicholas Eugenio
+description: ""
+date: "2023-08-30"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-08-30_pharmaverse_story"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Pharmaverse: from motivation to present
+
+**Human history and pharmaverse context**
```

Comment:
I think for titles we should use the title markdown to keep it consistent.
Please use: 

`# Human history and pharmaverse context`
Or
`## Human history and pharmaverse context`